src/org/jitsi/impl/neomedia/rtp/MediaStreamTrackDesc.java

```diff
@@ -374,6 +374,29 @@ public boolean matches(long ssrc)
     }
 
     /**
+     * Gets the maximum subjective quality index that complies to the max height
+     * specified as an argument.
+     *
+     * @param maxHeight the max height
+     * @return the maximum subjective quality index that complies to the max height
+     * specified as an argument, or 0.
```

Comment:
0 is a valid index though, isn't it? shouldn't it return -1 if it can't find anything?